### PR TITLE
Fixes the beach away mission.

### PR DESCRIPTION
**Pull request by Buggy123**
So we can have something for the gateway to lead to.

### CODE REVIEW

File: _maps/RandomZLevels/beach.dmm [hunk 1]
```diff
@@ -64,14 +64,18 @@
...
 "bn" = (/obj/structure/chair,/turf/simulated/floor/plating/beach/sand,/area/awaymission/beach)
-"bo" = (/turf/space,/area/awaymission/beach)
-"bp" = (/turf/indestructible{desc = "Very dense water, very difficult to pass."; icon = 'icons/misc/beach.dmi'; icon_state = "water"; name = "dense water"; opacity = 0},/area/space)
-"bq" = (/turf/simulated/floor/plating/beach/coastline,/area/space)
-"br" = (/turf/simulated/floor/plating/beach/coastline,/area/awaymission/beach)
-"bs" = (/turf/simulated/floor/plating/beach/coastline{density = 1; opacity = 1},/area/awaymission/beach)
-"bt" = (/turf/simulated/floor/plating/beach/water,/area/space)
-"bu" = (/turf/simulated/floor/plating/beach/water,/area/awaymission/beach)
-"bv" = (/turf/simulated/floor/plating/beach/water{density = 1; opacity = 1},/area/awaymission/beach)
+"bo" = (/turf/simulated/floor/holofloor/beach,/area/space)
```
Comment (KazeEspada) [inline _maps/RandomZLevels/beach.dmm]: bad bad bad wrong floor!

Comment (Buggy123) [inline _maps/RandomZLevels/beach.dmm]: Only one i could find, doesn't it work?

Comment (KazeEspada) [inline _maps/RandomZLevels/beach.dmm]: there should be the old plating one

Comment (Buggy123) [inline _maps/RandomZLevels/beach.dmm]: Ah, found the right one, fixing it...

**Comment by Buggy123** [comment]
Fixed.

**Comment by KazeEspada** [comment]
Where's the gateway?

**Comment by Buggy123** [comment]
Middle-west

**Pull request merged**